src/object/srv_obj.c

```diff
@@ -183,7 +183,6 @@ obj_rw_reply(crt_rpc_t *rpc, int status, uint64_t epoch,
 					if (sgls[i].sg_iovs[j].iov_buf == NULL)
 						continue;
```

Comment:
You can delete these two lines, and some of the {} braces as well.

Reply:
ok